kafka-gateway/README.md

```diff
@@ -18,9 +18,47 @@ example properties provided via VM options
 -Dspline.kafka.topic=spline-topic
 ```
 
-### Error recovery
-Currently, all received messages are acknowledged. Errors during message processing are logged. 
-The consumer is idempotent so if needed it can be rewind back, and the failed messages can be re-ingested. 
+### Idempotency
+In accord with Kafka best practices this consumer is indepmotent. 
+In practice, it means that one message can be consumed multiple times without risk of duplication. 
+Spline will recognize that the message was already consumed, and it will simply ignore it.
+
+### Scalability
+By default, Spline Consumer is using one Kafka Consumer. When need to scale arise you can run additional Spline Consumer instances.
+Same rules as for any other Kafka consumers apply.
```

Comment:
What is the meaning of the phrase?
Shouldn't it be "The same rules apply for the other Kafka consumers"?

Reply:
That there are some rules how Kafka consumers should be used, that I don't want to go into because it's all described in Kafka documentation. (For example, there is no point in having more consumers than partitions) Maybe it can be left out. We can just assume anybody who uses this already know how Kafka works.

Reply:
I removed the sentence.